### PR TITLE
Update legacy VM v1.4

### PR DESCRIPTION
**Pull request by camilbancioiu**
This PR updates the reference to the WASM VM v1.4.x, which contains many fixes for warm instances.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -12,7 +12,7 @@ require (
 	github.com/ElrondNetwork/elrond-vm-common v1.3.31
 	github.com/ElrondNetwork/wasm-vm-v1_2 v1.2.48
 	github.com/ElrondNetwork/wasm-vm-v1_3 v1.3.48
-	github.com/ElrondNetwork/wasm-vm-v1_4 v1.4.69
+	github.com/ElrondNetwork/wasm-vm-v1_4 v1.4.70-0.20221220091427-e43052768f7a
```
Comment (sasurobert) [inline go.mod:15]: proper release is needed

**Review by iulianpascalau — COMMENTED** [review]
just the proper release

File: integrationTests/testProcessorNode.go [hunk 2]
```diff
@@ -849,7 +849,7 @@ func (tpn *TestProcessorNode) createFullSCQueryService(gasMap map[string]map[str
 			EnableEpochsHandler: tpn.EnableEpochsHandler,
 			ArwenChangeLocker:   tpn.ArwenChangeLocker,
 			ESDTTransferParser:  esdtTransferParser,
-			Hasher:              tpn.Node.GetCoreComponents().Hasher(),
+			Hasher:              TestHasher,
```
Comment (iulianpascalau) [inline integrationTests/testProcessorNode.go:852]: :+1:

**Review by iulianpascalau — COMMENTED** [review]
just the proper release

File: go.mod (hunk 1 above)
Comment (camilbancioiu) [reply go.mod]: Done.

**Review by iulianpascalau — APPROVED** [review]

**Review by liviuf-ancas — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**